src/mpc-hc/CDarkTheme.cpp

```diff
@@ -342,7 +394,20 @@ void CDarkTheme::dbg(CString text, ...) {
     va_end(args);
 }
 
-void CDarkTheme::drawCheckBox(bool isChecked, bool isHover, bool useImage, CRect rectCheck, CDC *pDC) {
+UINT CDarkTheme::getResourceByDPI(CDC *pDC, const UINT *resources) {
+    int index;
+    int dpi = pDC->GetDeviceCaps(LOGPIXELSX);
+    if (dpi <= 96) index = 0;
+    else if (dpi <= 120) index = 1;
+    else if (dpi <= 144) index = 2;
+    else if (dpi <= 168) index = 3;
+    else if (dpi <= 196) index = 4;
+    else index = 4;
+
+    return resources[index];
+}

```

Comment:
Shouldn't this be:
if (dpi < 120) index = 0;
else if (dpi < 144) index = 1;
...
Otherwise too large image is used?

Reply:
Good idea, I'll make that change.  Smaller is better than too big.